src/firestore/delete.ts

```diff
@@ -248,23 +274,17 @@ export class FirestoreDelete {
       body = this.collectionDescendantsQuery(allDescendants, batchSize, startAfter);
     }
 
-    return api
-      .request("POST", "/v1beta1/" + url, {
-        auth: true,
-        data: body,
-        origin: api.firestoreOriginOrEmulator,
-      })
-      .then((res) => {
-        // Return the 'document' property for each element in the response,
-        // where it exists.
-        return res.body
-          .filter((x: any) => {
-            return x.document;
-          })
-          .map((x: any) => {
-            return x.document;
-          });
-      });
+    return this.apiClient.post("/v1/" + url, body).then((res) => {
```

Comment:
You can provide `apiVersion: "v1"` to the Client constructor above, if you wish.

Also here to say: this is an intentional change to `v1`, right?

Reply:
Yes this is intentional, and good to know!